### ISSUE TITLE
Cross-termporal forecasting 

### ISSUE THREAD

**Issue by ncooder** [Nixtla/hierarchicalforecast#282]
### Description

Add the cross- and termporal forecasting based on R packages:
[thief](https://github.com/robjhyndman/thief)
[MAPA](https://github.com/trnnick/mapa)
[FoReco](https://github.com/daniGiro/FoReco)

**Comment by elephaint** [comment]
@ncooder Thanks for the suggestion, you are very right for suggesting this - it's my top priority of next feature to add.

**Comment by christopher-titchen** [comment]
@ncooder @elephaint I agree, adding temporal and cross-temporal forecast reconciliation would be a nice feature, especially for long-range forecasts.

**Comment by marcozanotti** [comment]
Hi,  
will THIEF and MAPA be available soon in Nixtla? Those are really breaking methods sometimes.
Thanks for all your work.

**Comment by elephaint** [comment]
> Hi, will THIEF and MAPA be available soon in Nixtla? Those are really breaking methods sometimes. Thanks for all your work.

Target is next two months.

**Comment by wmjones** [comment]
Very excited for this feature keep up the amazing work!

**Comment by seth602** [comment]
Also very excited for this feature!

**Comment by anthonygiorgio97** [comment]
Any news on when this feature will.be available?
